bears/general/IndentationBear.py

```diff
@@ -93,19 +97,29 @@ def get_indent_levels(self, file, filename, indent_types, annotation_dict):
         :return:                A tuple containing the levels of indentation of
                                 each line.
         """
+        ranges = []
+        for indent_specifier in indent_types:
+            ranges += self.get_specified_block_range(
+                                    file,
+                                    filename,
+                                    indent_specifier,
+                                    indent_types[indent_specifier],
+                                    annotation_dict)
+
+        ranges = sorted(ranges, key=lambda x: x.start.line)
         indent_levels = []
-        ranges = self.get_specified_block_range(
-                         file, filename, indent_types, annotation_dict)
         indent, next_indent = 0, 0
         for line in range(0, len(file)):
             indent = next_indent
             for _range in ranges:
                 if _range.start.line == line + 1:
                     next_indent += 1
+
                 if(_range.end.line == line + 1 and
                    (file[line].lstrip()[0] in indent_types.values())):
                     indent -= 1
                     next_indent -= 1
+
```

Comment:
empty lines! yeay, love them :smile: 

no, really.

but baaaasically, if you would insert new lines in an own commit, I would just ack that very quickly and we were done with it. This way it kinda bloats your commit.

You don't need to remove them now from this commit, but generally, splitting your changes into a lot of atomic commits can make things easier and simpler. Like with the coalang stuff at the top of this file.

I'm not afraid to review a PR with lot's of commits... I'm afraid of a PR with gigantic commits that take an hour to review each :smile: 

your new lines here are not that bad, but it needed to be told anyways :D